packages/angular/projects/ui-angular/src/lib/components/authenticator/components/setup-totp/setup-totp.component.ts

```diff
@@ -44,15 +43,12 @@ export class SetupTotpComponent implements OnInit {
     // TODO: This should be handled in core.
     const state = this.authenticator.authState;
     const actorContext = getActorContext(state) as SignInContext;
-    const { user } = actorContext;
+    const { user, formFields, country_code } = actorContext;
```

Comment:
What are we `country_code` for? Not seeing it referenced again in this file?

Reply:
Deleted

Reply:
#b226db3